docs/UserGuide.md

```diff
@@ -244,4 +250,4 @@ Action | Format, Examples
 **Tag Delete**| `tag delete INDEX` <br> e.g., `tag delete 1` 
 **Tag Edit**| `tag edit INDEX t/TAG` <br> e.g., `tag edit 1 t/CS2101` 
 **Tag List** | `tag list`
-**Tag View** | `tag view INDEX`<br> e.g., `tag view 3`
+**Tag Find** | `tagFind  KEYWORD [MORE_KEYWORDS]`<br> e.g., `tagfind friends colleagues`
```

Comment:
so is the command tagFind or tagfind?

Reply:
It is supposed to be `tagfind`. Will open another issue to change the other commands to lowercase.